entities/.happy/terraform/envs/dev/main.tf

```diff
@@ -47,7 +47,6 @@ module "stack" {
   }
   additional_env_vars = {
     AWS_REGION              = "us-west-2"
-    BOTO_ENDPOINT_URL       = "http://motoserver.czidnet:4000"
```

Comment:
I'm not that familiar with how our infra is set up, is there a chance that we will need moto set up for our remote dev instances?

I think we'll also need to update our downstream code that reads from these settings, here's an example of where we get our s3 client based on the `BOTO_ENDPOINT_URL`: https://github.com/chanzuckerberg/czid-platformics/blob/8e11b9dae9903242c1b7ad30645afc39a7a3a730/platformics/api/core/deps.py#L126

Reply:
In remote dev we should still be using real AWS. Interesting on the settings piece, it seems that workflows functions without this config. I think this is fine because the pydantic schema declares this setting as optional so it will pass `None` into here which defaults to real AWS.